wiki/pt/pt-Installation-for-Linux.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: wiki
+title: "Installation for Linux"
+lang: "pt"
+permalink: "/wiki/Installation-for-Linux"
+---
+
+{% include breadcrumb.html root="Usand Jamulus" branch1="Começando" branch1-url="Getting-Started" %}
+
+# Instalação no Linux
+Primeiro tenha certeza que você já leu a página [Começando](Getting-Started)
+
+### Debian e Ubuntu
+
+1. Baixe o [pacote .deb atual]({{ site.download_root_link }}{{ site.download_file_names.deb-gui }})
+1. Atualize o apt: `sudo apt-get update`
```

Comment:
Could you please check the English translation? I think this is not based on @gilgongo 's changes which were included later on. I think we'll have a process which is a lot simpler soon: https://github.com/jamulussoftware/jamulus/discussions/1725

Reply:
It's based on the 'translate3_8_0' branch. Same for the DE file. @ewarning there were some edits made to this document that due to a confusion didn't get merged from 'changes' to 'translate'. It would be great if you could edit them to follow the 'changes' version of en-Installation-for-Linux.md. - it's not that much. But if it's too much hassle don't worry as they aren't major changes.